packages/wonder-blocks-breadcrumb/components/breadcrumb.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import type {AriaProps} from "@khanacademy/wonder-blocks-core";
+import {addStyle} from "@khanacademy/wonder-blocks-core";
+import BreadcrumbItem from "./breadcrumb-item.js";
+
+type Props = {|
+    ...AriaProps,
+
+    /**
+     * This is the content for the collection of Breadcrumbs
+     */
+    children: React.ChildrenArray<React.Element<typeof BreadcrumbItem>>,
+|};
+
+const StyledList = addStyle("ol");
+
+/**
```

Comment:
Can you expand this comment and start with an explanation of what a Breadcrumb is and where we expect people to use it? That way when people read the docs they'll have a bit more context.